src/firefly/js/visualize/DrawLayerCntlr.js

```diff
@@ -287,27 +301,39 @@ export function dispatchDetachLayerFromPlot(id,plotId, detachPlotGroup=false,
 }
 
 /**
- * @summary Create plot layer containing the regions based on region file or region description
- * @param drawLayerId required
- * @param layerTitle  layerTitle is set based on drawLayerId or default setting it is unset
- * @param fileOnServer
- * @param regionAry
- * @param plotId The region layer is created on all plots of the active plot group in plotId is empty
- * @param dispatcher
+ * @summary Create drawing layer based on region file or region description
+ * @param {string} drawLayerId id of the drawing layer to be created, required
+ * @param {string} layerTitle  if it is empty, it will be created internally
+ * @param {string} fileOnServer region file name on server
+ * @param {string[]|string} regionAry  array or string of region description
+ * @param {string[]|string} plotId  array or string of plot id. If plotId is empty, all plots of the active group are applied.
+ * @param {function} dispatcher
```

Comment:
should be last param

In general, there quite a few eslint warnings/errors (with our .eslintrc) in the changed files, like unused imports, vars, etc. It would be beneficial to fix them.